kernel/kernel-api/src/test/scala/io/delta/kernel/test/MockTableClientUtils.scala

```diff
@@ -76,7 +76,7 @@ trait MockTableClientUtils {
 /**
  * Base class for mocking [[JsonHandler]]
  */
-trait BaseMockJsonHandler extends JsonHandler {
+trait BaseMockJsonHandler extends JsonHandler with MockTableClientUtils {
```

Comment:
why is the `with MockTableClientUtils` is needed?

Reply:
Removed.